meta/send.rb

```diff
@@ -379,8 +379,6 @@
   mutation 'foo.public_send(:bar, -1, two: true, **kwargs, &block)'
   mutation 'foo.public_send(:bar, 2, two: true, **kwargs, &block)'
   mutation 'foo.public_send(:bar, two: true, **kwargs, &block)'
-  mutation 'foo.public_send(:bar, 1, nil => true, **kwargs, &block)'
-  mutation 'foo.public_send(:bar, 1, self => true, **kwargs, &block)'
```

Comment:
nice :)

Reply:
@dgollahon The reason this was not found before is that argument promotion only happens on one argument.

So we had to find a single kwarg to see it.

Mutants corpus test is quite heavy and so far it "only" generates the mutation, but does NOT unparse + parse (full validation).

With mutant now being able to natively use multiple processes (The parallel engine is generic) this can be fixed.

I'll not use `ractor` as its experimental, and it'll take a long time till the last Ruby that does not have `ractor` will be EOL.

Reply:
Ahh, full parse/unparse validation would be great to add.